os/arch/arm/src/amebasmart/amebasmart_pmc.c

```diff
@@ -57,11 +57,13 @@
 #include "FreeRTOS.h"
```

Comment:
Please update the description of this commit to briefly explain the changes made in the commit

Reply:
done~